web/src/data/songs_of_the_week.js

```diff
@@ -0,0 +1,59 @@
+import { createDate } from "utils/date_utils";
+import RileyHeadshot from "assets/images/riley.jpg";
+
+const dayjs = require("dayjs");
+
+class SongOfTheWeek {
+  constructor(startDate, quoteInfo, songInfo, songLinks) {
+    this.startDate = startDate;
+    this.quoteInfo = quoteInfo;
+    this.songInfo = songInfo;
+    this.songLinks = songLinks;
+  }
+}
+
+const attributors = {
+  riley: {
+    name: "Riley",
+    picture: RileyHeadshot,
+  },
+};
+
+const songs = [
+  new SongOfTheWeek(
+    createDate("05/26/2024"),
+    {
+      quote:
+        "This song is one of my favorites to dance to. " +
+        "This month's creative expressions class will be working on choreography to this song. " +
+        "Absolute banger, and brings back a lot of great memories.",
+      attributionInfo: attributors.riley,
+    },
+    {
+      name: "Old School Turntables",
+      artist: "Matt DiMona",
+      year: "2017",
```

Comment:
any way we could pull this from the Spotify API instead of hardcoding it here?

Reply:
feel free to take a look at https://developer.spotify.com/documentation/web-api

TLDR is not simple. Way easier to just hardcode

Reply:
u right